	Here we need AC_CHECK_TOOL, not AC_CHECK_TARGET_TOOL!

	AC_CHECK_TARGET_TOOL is, e.g., for a compiler built on BUILD,
	executing on HOST and creating executable code for TARGET.


diff -ur icu-49.rc.orig/source/configure.in icu-49.rc/source/configure.in
--- icu-49.rc.orig/source/configure.in	2012-03-02 23:14:56.000000000 +0100
+++ icu-49.rc/source/configure.in	2012-03-03 17:20:02.000000000 +0100
@@ -354,7 +354,7 @@
 AC_PROG_RANLIB
 
 # look for 'ar' the proper way
-AC_CHECK_TARGET_TOOL(AR, ar, false)
+AC_CHECK_TOOL(AR, ar, false)
 
 AC_MSG_CHECKING([whether to enable renaming of symbols])
 enabled=yes
